app/docs/0.10.x/secure-admin-api.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Securing the Admin API
+---
+
+# Overview
+
+Kong's Admin API provides a RESTful interface for administration and
+configuration of APIs, plugins, consumers, and credentials. Because this
+API allows full control of Kong, it is important to secure this API against
+unwanted access. This document touches on a few approaches to take to secure
+the Admin API.
+
+## Table of Contents
+
+- [Network Layer Access Restrictions](#network-layer-access-restrictions)
+  - [Reduce Listening Footprint](#reduce-listening-footprint)
+  - [Layer 3/4 Network Controls](#layer-34-network-controls)
+- [Kong API Loopback](#kong-api-loopback)
+- [Custom Nginx Configuration](#custom-nginx-configuration)
+
+## Network Layer Access Restrictions
+
+### Reduce Listening Footprint
+
+By default, Kong will accept requests for both the public-facing entrypoint, and
+the Admin API, on `0.0.0.0`, which will bind to all available interfaces on the
+host. Reducing this exposure footprint by limiting the interfaces by which the
+Admin API can be accessed is a foundational step. This setting can be adjusted
+via the `admin_listen` Kong configuration directive. For example:
+
+`admin_listen 127.0.0.1:8001`
+
+This will define the Nginx `listen` directive used by Kong to the prescribed
+value, instructing Kong to only respond to requests received on the localhost
+interface.
+
+[Back to TOC](#table-of-contents)
+
+### Layer 3/4 Network Controls
+
+In cases where the Admin API must be exposed beyond a localhost interface,
+network security best practices dictate that network-layer access be restricted
+as much as possible. Consider an environment in which Kong listens on a private
+network interface, but should only be accessed by a small subset of an IP range.
+In such a case, host-based firewalls (e.g. iptables) are useful in limiting
+input traffic ranges. Consider the following example:
```

Comment:
"Consider..." starts two sentences in the same paragraph - consider changing one of them to something other than "Consider"